models/src/main/java/com/vimeo/networking2/Album.kt

```diff
@@ -0,0 +1,105 @@
+package com.vimeo.networking2
+
+import com.vimeo.networking2.enums.AlbumLayoutType
+import com.vimeo.networking2.enums.AlbumThemeType
+import com.vimeo.networking2.enums.SortType
+import java.util.*
+
+/**
+ * Group of videos to share publicly or privately.
+ */
+data class Album(
+
+    /**
+     * Hexadecimal color code for the decorative color.
+     * For example, album videos use this color for player buttons.
+     */
+    val brandColor: String? = null,
+
+    /**
+     * The time in ISO 8601 format that the album was created.
+     */
+    val createdTime: Date? = null,
+
+    /**
+     * A brief description of the album's content.
+     */
+    val description: String? = null,
+
+    /**
+     * The total duration in seconds of all the videos in the album.
+     */
+    val duration: Int? = null,
+
+    /**
+     * Embed data for the album.
+     */
+    val embed: AlbumEmbed? = null,
+
+    /**
+     * Whether to hide the Vimeo navigation when viewing the album.
+     */
+    val hideNav: Boolean? = null,
+
+    /**
+     * The album's layout preference.
+     */
+    val layout: AlbumLayoutType? = null,
+
+    /**
+     * The URL to access the album.
+     */
+    val link: String? = null,
+
+    /**
+     * Metadata about the album.
+     */
+    val metadata: Metadata<AlbumConnections, AlbumInteractions>? = null,
+
+    /**
+     * The time in ISO 8601 format when the album was last modified.
+     */
+    val modifiedTime: Date? = null,
+
+    /**
+     * The album's display name.
+     */
+    val name: String? = null,
+
+    /**
+     * The active image for the album; defaults to the thumbnail of the last
+     * video added to the album.
+     */
+    val pictures: PictureCollection? = null,
```

Comment:
In the current implementation this is an array of PictureCollection objects. The API returns the thumbnails for the first three videos in the album. It is something that may vary across API versions though. Hopefully in the future it will return to being a single PictureCollection object.

Reply:
I found the current implementation return a single `PictureCollection` object. The `PictureCollection` object itself has an array of `Picture` objects which correspond to the thumbnails.